oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/H2StorageProvider.java

```diff
@@ -131,6 +135,9 @@ public void prepare() throws ServiceNotProvidedException, ModuleStartException {
 
     @Override
     public void start() throws ServiceNotProvidedException, ModuleStartException {
+        MetricsCreator metricCreator = getManager().find(TelemetryModule.NAME).provider().getService(MetricsCreator.class);
+        GaugeMetrics healthChecker = metricCreator.createHealthCheckerGauge("storage_h2", MetricsTag.EMPTY_KEY, MetricsTag.EMPTY_VALUE);
+        healthChecker.setValue(1);
```

Comment:
> Why always `1`? According to your doc, 1 should mean unhealthy? I think you missed some codes.

This may be the initial status "unhealthy", and after the connection is established (line 142-145), the value should be set to `0`, so I reckon the line at 149 should be `healthChecker.setValue(0);`? @hanahmily

Reply:
Fixed with an asyn health check scheduler.